static/style.css

```diff
@@ -121,11 +129,33 @@ button, select {
   cursor: pointer;
 }
 
-input:hover, button:hover, select:hover {
+#input-wrapper {
+  padding: 0;
+  flex-grow: 1;
+  cursor: unset;
+  display: flex;
+  align-items: center;
+}
+
+#input-wrapper * {
+  background: unset;
+  border: unset;
+  height: 100%;
+}
+
+#input-wrapper svg {
+  transform: scale(1.2);
+}
+
+#input-wrapper button:hover {
+  color: var(--secondary);
+}
+
+input:hover, button:hover, select:hover, #input-wrapper:hover {
   border-bottom-color: var(--secondary);
 }
 
-input:focus, button:focus, select:focus {
+input:focus, button:focus, select:focus, #input-wrapper:focus-within {
```

Comment:
This okay? https://caniuse.com/?search=focus-within
![image](https://user-images.githubusercontent.com/45497981/199292157-db0a6d60-2f9e-481d-9093-0eb3aff2b255.png)

Reply:
owo.vc already uses the fetch API for web requests, so supporting IE isn't a huge concern right now.